arviz/stats/stats.py

```diff
@@ -786,7 +795,9 @@ def summary(
             if len(values.shape[1:]):
                 metric = list(values.metric.values)
                 data_dict = {}
-                for idx in np.ndindex(values.shape[1:]):
+                for idx in np.ndindex(values.shape[1:] if order == "C" else values.shape[1:][::-1]):
```

Comment:
What does the if statement do here?

Reply:
Small numpy indexing trick to use 'F' order.

- C = iterate last dim first
- F = iterate first dim first

Reply:
But in this statement if `order =="C"` goes nowhere? Or in other words if `order == "C"` it doesn't look like there's any code that gets executed?

Reply:
Ah, conditional expression https://docs.python.org/3/reference/expressions.html#conditional-expressions

`values.shape[1:] if order == "C" else values.shape[1:][::-1]`

`value_bool_true if bool else value_bool_false`

you could also chain them

`value1 if bool else value2 if bool2 else value3 if ....`